src/main/java/nextstep/ladder/model/Ladder.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.ladder.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Ladder {
+    private final List<Line> lines = new ArrayList<>();
+
+    public Ladder(int height, int countOfPerson) {
+        validateHeight(height);
+        validateCountOfPerson(countOfPerson);
+        IntStream.range(0, height)
+                .forEach(i -> lines.add(new Line(countOfPerson)));
+    }
```

Comment:
```suggestion
    private final List<Line> lines;

    public Ladder(int height, int countOfPerson) {
        validateHeight(height);
        validateCountOfPerson(countOfPerson);
        this.lines = IntStream.range(0, height)
                .mapToObj(i -> new Line(countOfPerson))
                .collect(Collectors.toList());
    }
```
여기도 바로 초기화해줄 수 있습니다 :)